packages/datadog-instrumentations/src/oracledb.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const connectionAttributes = new WeakMap()
+const poolAttributes = new WeakMap()
+
+const executeStartChannel = channel('apm:oracledb:execute:start')
+const executeErrorChannel = channel('apm:oracledb:execute:error')
+const executeEndChannel = channel('apm:oracledb:execute:end')
```

Comment:
Let's use `finish` instead to be inline with what Node does and also avoid confusion with `end` vs `async-end`.